CHANGELOG.md

```diff
@@ -18,6 +18,72 @@ Valid subsections within a version are:
 
 Things to be included in the next release go here.
 
+### Added
+
+- Added curve query support for the MSO2KB series scopes
+
+---
+
+## v2.2.2 (2024-08-14)
```

Comment:
It looks like this was added on accident.

Reply:
This shows you adding the v2.2.2 changes, but they are already in the file. The only changes that should show up are the ones in the unreleased section. You need to delete the duplicate sections. It should be obvious if you look in the file. There are duplicate sections that should not be there.

Reply:
Everything from line 23 to line 83 needs to be removed. It is duplicating the sections that come next.